developing-with-nats/security/token.md

```diff
@@ -50,15 +50,15 @@ let nc = NATS.connect({url: `nats://127.0.0.1:${port}`, token: "mytoken!"});
 ```python
 nc = NATS()
 
-await nc.connect(servers=["nats://mytoken@demo.nats.io:4222"])
+await nc.connect(servers=["nats://demo.nats.io:4222"], token="mytoken")
```

Comment:
I would add in the documentation above, that "Tokens are supported as a rudimentary security mechanism but we **highly** recommend using one of the other NATS authentication mechanisms."

Reply:
added, also added picking the right authentication scheme to the google doc